routers/common/repo.go

```diff
@@ -104,3 +106,34 @@ func ServeData(ctx *context.Context, name string, size int64, reader io.Reader)
 	_, err = io.Copy(ctx.Resp, reader)
 	return err
 }
+
+func ServeLargeFile(ctx *context.Context, name string, time time.Time, reader io.ReadSeeker) error {
+	name = strings.ReplaceAll(name, ",", " ")
+	ctx.Resp.Header().Set("Cache-Control", "public,max-age=86400")
```

Comment:
We shouldn't store in public caches if the gitea instance is private. Also, 1 day seems like a long period, I suggest just specifying `max-age=300` only like GH does.

Reply:
need help

> As you may have noticed it does not fail because of your code. Could you please stop with these commits until someone could have a look?

Okay~ the build check is too slow to continue.